internal/model/model.go

```diff
@@ -1774,11 +1774,21 @@ func (m *Model) CommitConfiguration(from, to config.Configuration) bool {
 	return true
 }
 
-func symlinkInvalid(isLink bool) bool {
-	if !symlinks.Supported && isLink {
-		SymlinkWarning.Do(func() {
+func symlinkInvalid(folder string, fi db.FileIntf) bool {
+	if !symlinks.Supported && fi.IsSymlink() && !fi.IsInvalid() && !fi.IsDeleted() {
+		symlinkWarning.Do(func() {
 			l.Warnln("Symlinks are disabled, unsupported or require Administrator privileges. This might cause your folder to appear out of sync.")
```

Comment:
I was actually going to do that. But then it broke my heart to add another reference to the fucking logs so I didn't. But maybe we should anyway. :disappointed: